VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimStatus/ClaimStatus.tsx

```diff
@@ -49,43 +49,32 @@ function ClaimStatus({ claim, claimType }: ClaimStatusProps) {
     // alternative check if need to update: isClosedClaim = claim.attributes.decisionLetterSent && !claim.attributes.open
     const isActiveClaim = claimType === ClaimTypeConstants.ACTIVE
 
-    const whyWeCombineOnPress = () => {
-      logAnalyticsEvent(Events.vama_claim_why_combine(claim.id, claim.attributes.claimType, claim.attributes.phase))
-      navigateTo('ConsolidatedClaimsNote')
-    }
-
-    const whatShouldOnPress = () => {
-      logAnalyticsEvent(Events.vama_claim_disag(claim.id, claim.attributes.claimType, claim.attributes.phase))
-      navigateTo('WhatDoIDoIfDisagreement', {
-        claimID: claim.id,
-        claimType: claim.attributes.claimType,
-        claimStep: claim.attributes.phase,
-      })
-    }
-
     if (isActiveClaim) {
-      const detailsFAQListItems: Array<SimpleListItemObj> = [
-        {
-          text: t('claimDetails.whyWeCombine'),
-          onPress: whyWeCombineOnPress,
-          testId: a11yLabelVA(t('claimDetails.whyWeCombine')),
-        },
-        {
-          text: t('claimDetails.whatShouldIDoIfDisagree'),
-          onPress: whatShouldOnPress,
-          testId: a11yLabelVA(t('claimDetails.whatShouldIDoIfDisagree')),
-        },
-      ]
-
       // TODO: determine when showCovidMessage prop for EstimatedDecisionDate would be false
 
       return (
         <Box>
-          {claim && <ClaimTimeline attributes={claim.attributes} claimID={claim.id} />}
+          {claim && featureEnabled('claimPhaseExpansion') && (
```

Comment:
I could honestly think at this point it doesn't really matter cause all it would do is shift around where in the files we would have this check and maybe save us a few minutes when cleaning up. I think in the future as long as we have the feature flag/release ASAP stuff outlined before hand from the beginning we can architect this much better throughout the process.